candle-transformers/src/models/llava/config.rs

```diff
@@ -0,0 +1,300 @@
+use std::collections::HashMap;
+
+use crate::models::{
+    clip::{text_model::Activation, vision_model::ClipVisionConfig},
+    llama::Config,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::models::llava::clip_image_processor::CLIPImageProcessor;
+
+// original config from liuhaotian/llava
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct LLaVAConfig {
+    pub _name_or_path: String,
+    pub architectures: Vec<String>,
+    //pub attention_bias: bool,
```

Comment:
Better not to keep the commented code even if they indicate some fields of the original config, they are likely to get out of date pretty quickly.

Reply:
Ok I remove them